.github/workflows/ci.yml

```diff
@@ -0,0 +1,67 @@
+name: CI
+
+on: [
+        push,
+        workflow_dispatch
+]
+
+jobs:
```

Comment:
* Copy the content from https://github.com/spryker-eco/loggly/blob/master/.github/workflows/ci.yml.